### PR TITLE
Template labels

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: openshift/buildConfig-template.yaml [hunk 1]
```diff
@@ -13,14 +10,20 @@ metadata:
     template.openshift.io/documentation-url: https://github.com/Thoth-Station/
     template.openshift.io/long-description: This template defines resources needed to deploy Thoth Graph Refresh Job as a Proof-of-Concept to OpenShift.
     template.openshift.io/provider-display-name: Red Hat, Inc.
+  labels:
+    template: graph-refresh-buildconfig
+    thoth: 0.1.0
+    app: thoth
```
Comment (goern) [inline openshift/buildConfig-template.yaml]: here you keep the version ;)

**Review by goern — DISMISSED** [review]
closes #51

File: openshift/buildConfig-template.yaml (hunk 1 above)
Comment (fridex) [reply openshift/buildConfig-template.yaml]: Not anymore!

**Comment by fridex** [comment]
recheck

**Pull request merged**